### DISCUSSION TITLE
run lsposed instead of shortcut

### DISCUSSION THREAD

**Discussion by macenjie** [LSPosed/LSPosed#1257] [Q&A] [answered]
Can I run lsposed framework by adb shell or other ways instead of the shortcut on the desktop.
If the shortcut is deleted accidentally, Can I recover it any way?

**Comment by yujincheng08** [comment, marked as answer]
dial `*#*#5776733#*#*`

**Reply by ghost** [reply]
Not available with version 1.6.0 but from [this](https://github.com/LSPosed/LSPosed/actions/runs/1319911903)

**Reply by xerta555** [reply]
> 
> 
> dial _#_#5776733#_#_

I just try that, it won't works.

**Reply by furopi** [reply]
> dial `*#*#5776733#*#*`

I bookmarked this page because it's so useful! Thank you so much!

**Reply by victories** [reply]
It works. Android 13 & miui 14 & LSPosed-v1.8.6-6712-zygisk-release

**Reply by xerta555** [reply]
> It works. Android 13 & miui 14 & LSPosed-v1.8.6-6712-zygisk-release

For as long as it's been supposed to work (not counting all the patches to get it right) thank goodness it does 😁

**Reply by Jackboy001** [reply]
Hello, there's no dialer on Android tablet. Is there any other way to open it?

**Reply by victories** [reply]
You can install the software from playstore https://play.google.com/store/apps/details?id=org.lsposed.manager

**Comment by macenjie** [comment]
I tryed  adb shell am start -a android.intent.action.CALL -d tel:##5776733##
not correct

**Reply by yujincheng08** [reply]
no adb shell. dial it in ur dialer.

**Comment by macenjie** [comment]
I know, it's android_secret_code.  dialer manually is also not working.
The lsposed apk file is released by 12 days ago, Need update?

[core] Add secret code receiver  - Howard20181 committed 4 days ago

**Comment by spiral009** [comment]
This dial ain't doin nothin

**Comment by rophiroth** [comment]
Same problem on oneplus 9 latest build (security patch)

**Comment by furopi** [comment]
So after all it stopped working 😞
I finally managed to get the manager up and running again. It was hours of work... The solution I found was to use the app "Unnotification" to make the notification visible again and then restore the shortcut from the settings of the manager. Unnotification was once in the Play Store, but sadly isn't anymore... But the APK can be found on sites like apkpure! Hope this helps at least somebody! 

Cheers! 🫡
Florian
